src/Main.hs

```diff
@@ -61,6 +61,7 @@ main = do
 runHdfs :: forall a. Hdfs a -> IO a
 runHdfs hdfs = handle exitError $ do
     config <- getConfig
+    print config
```

Comment:
You might want to remove this line before merging J
(I have this running on my Mac via ssh socks proxy, works fine but prints the config every time.)